package.json

```diff
@@ -40,8 +40,8 @@
   },
   "devDependencies": {
     "@prisma-labs/prettier-config": "0.1.0",
-    "@prisma/client": "3.14.0",
-    "@prisma/sdk": "3.14.0",
+    "@prisma/client": "^3.15.0",
+    "@prisma/sdk": "^3.15.0",
```

Comment:
```suggestion
    "@prisma/client": "3.15.0",
    "@prisma/sdk": "3.15.0",
```

Reply:
We use renovate to manage dev deps :)

Reply:
Yes, but the first thing Renovate would do is to add a PR to pin the exact version (see the state before).